### ISSUE TITLE
"Hide the 'Love this site?' module" option hides the related question module as well

### ISSUE THREAD

**Issue by ooker777** [soscripted/sox#384]
**Describe the bug**
"Hide the 'Love this site?' module" option hides the related question module as well

**To Reproduce**
Steps to reproduce the behaviour:
1. Check the "Hide the 'Love this site?' module" option 
2. The related question module is hidden as well

**SOX errors logged in console**
```
is-it-possible-to-adhere-to-the-practices-and-beliefs-of-both-buddhist-and-chris:4 Active resource loading counts reached a per-frame limit while the tab was in background. Network requests will be delayed until a previous loading finishes, or the tab is brought to the foreground. See https://www.chromestatus.com/feature/5527160148197376 for more details
chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4 ERROR: Execution of script 'AutoTOC' failed! GM_registerMenuCommand is not defined
(anonymous) @ chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4
chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4 ReferenceError: GM_registerMenuCommand is not defined
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:207)
    at Window.tms_b38c2573_1e60_4ada_afe4_a27fb89861ef (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:210)
    at <anonymous>:2:494
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4)
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:5)
    at Object.eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:211)
    at <anonymous>:2:494
    at Object.E_c (<anonymous>:3:306)
    at ia (eval at exec_fn (is-it-possible-to-adhere-to-the-practices-and-beliefs-of-both-buddhist-and-chris:1), <anonymous>:63:203)
    at Object.create (eval at exec_fn (is-it-possible-to-adhere-to-the-practices-and-beliefs-of-both-buddhist-and-chris:1), <anonymous>:75:51)
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  logging sox stored values --- 
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  access token set
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-addSBSBtn-settings
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {"sbsByDefault":false}
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-debug
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  true
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-lastVersionInstalled
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  2.4.3DEV
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOXSETTINGS
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-showTagWikiLinkOnTagPopup","Appearance-hideWelcomeBackMessage","Appearance-hideHowToAskWhenZoomed","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-hideHotNetworkQuestions","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Extras-alwaysShowImageUploadLinkBox","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-sortByBountyAmount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue","Extras-addOnTopicLinkToSiteSwitcher","Extras-customMagicLinks"]
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  downvotedPostsEditAlert
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {"112350":{"questionId":"112350","addedDate":1531107411248,"sitename":"academia","lastCheckedTime":1532447213778}}
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  editReasons
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  [{"formatting":"Improved Formatting"},{"spelling":"Corrected Spelling"},{"grammar":"Fixed grammar"},{"greetings":"Removed thanks/greetings"},{"tag":"Added tags"}]
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  metaNewQuestionAlert-lastQuestions
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {}
is-it-possible-to-adhere-to-the-practices-and-beliefs-of-both-buddhist-and-chris:1 [Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: https://use.fontawesome.com/releases/v5.1.0/webfonts/fa-solid-900.woff2
secure.quantserve.com/quant.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:7 [Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: https://use.fontawesome.com/releases/v5.1.0/webfonts/fa-regular-400.woff2
```

**Expected behaviour**
Hide the 'Love this site?' module, but keep the related question module visible

**Environment**
SOX version: 2.4.3DEV
Platform: Tampermonkey

---

### Features Enabled 

    ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-showTagWikiLinkOnTagPopup","Appearance-hideWelcomeBackMessage","Appearance-hideHowToAskWhenZoomed","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideHireMe","Sidebar-hideHotNetworkQuestions","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Extras-alwaysShowImageUploadLinkBox","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-sortByBountyAmount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue","Extras-addOnTopicLinkToSiteSwitcher","Extras-customMagicLinks"]

**Comment by shu8** [comment]
Thanks @ooker777, I'll get this fixed soon :)

**Comment by shu8** [comment]
@ooker777 I can't reproduce this -- are we talking about this section of the sidebar:?

![image](https://user-images.githubusercontent.com/8850830/54160904-a484a000-4448-11e9-82a2-030c748cb3b8.png)

If so, could you check to see if this still happens, and if it does is it only on a specific site?

**Comment by ooker777** [comment]
Yes, this section. For example, it is hidden when the option "Hide the 'Love this site?' module" is checked in [this question](https://writing.stackexchange.com/q/43305/11428)

**Comment by shu8** [comment]
Could you screenshot the 'love this site' module? I've got the feature disabled and can't even see it on any site!

Are you logged in when this happens?

**Comment by ooker777** [comment]
This is weird, because the bug seems to affect on different sites on another machine and/or browser. [This question on Psychology](https://psychology.stackexchange.com/questions/21790/what-are-the-methodological-differences-between-a-constructed-and-a-standardi) has no effect if you check the option, the result is always like this:

 ![](https://i.imgur.com/BjhhrYF.png)

However, on [the question on Writing](https://writing.stackexchange.com/questions/42125/how-to-describe-skin-colour-if-white-is-not-the-point-of-reference),

- Uncheck the box will show all sections 
 ![](https://i.imgur.com/3NlVG2b.png)
- Check the box will hide not only the related question section, but also the stat one! That is, everything is hidden!
 ![](https://i.imgur.com/pIaQ7iE.png)

The browser now is Vivaldi (the previous report is on Firefox Quantum). I use Tempermonkey on both, and has disable all other scripts except SOX.

# Console log

```
how-to-describe-skin-colour-if-white-is-not-the-point-of-reference:1 Active resource loading counts reached a per-frame limit while the tab was in background. Network requests will be delayed until a previous loading finishes, or the tab is brought to the foreground. See https://www.chromestatus.com/feature/5527160148197376 for more details
secure.quantserve.com/quant.js:1 Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4 ERROR: Execution of script 'AutoTOC' failed! GM_registerMenuCommand is not defined
(anonymous) @ chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4
chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4 ReferenceError: GM_registerMenuCommand is not defined
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:207)
    at Window.tms_b38c2573_1e60_4ada_afe4_a27fb89861ef (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:210)
    at <anonymous>:2:494
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:4)
    at eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:5)
    at Object.eval (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?id=b38c2573-1e60-4ada-afe4-a27fb89861ef:211)
    at <anonymous>:2:494
    at Object.E_c (<anonymous>:3:306)
    at ia (eval at exec_fn (how-to-describe-skin-colour-if-white-is-not-the-point-of-reference:1), <anonymous>:63:272)
    at Object.create (eval at exec_fn (how-to-describe-skin-colour-if-white-is-not-the-point-of-reference:1), <anonymous>:75:51)
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  logging sox stored values --- 
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  access token set
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-addSBSBtn-settings
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {"sbsByDefault":false}
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-debug
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  true
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOX-lastVersionInstalled
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  2.4.3DEV
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  SOXSETTINGS
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  ["Appearance-addAuthorNameToInboxNotifications","Appearance-alignBadgesByClass","Appearance-answerTagsSearch","Appearance-colorAnswerer","Appearance-highlightQuestions","Appearance-localTimestamps","Appearance-markEmployees","Appearance-metaChatBlogStackExchangeButton","Appearance-spoilerTip","Appearance-standOutDupeCloseMigrated","Appearance-tabularReviewerStats","Appearance-topAnswers","Appearance-unspoil","Appearance-addTimelineAndRevisionLinks","Appearance-showTagWikiLinkOnTagPopup","Appearance-hideWelcomeBackMessage","Appearance-hideHowToAskWhenZoomed","Comments-autoShowCommentImages","Comments-commentReplies","Comments-commentShortcuts","Comments-confirmNavigateAway","Comments-copyCommentsLink","Comments-moveBounty","Comments-showCommentScores","Comments-hiddenCommentsIndicator","Editing-addSBSBtn","Editing-editComment","Editing-editReasonTooltip","Editing-findAndReplace","Editing-kbdAndBullets","Editing-titleEditDiff","Editing-inlineEditorEverywhere","Flags-flagOutcomeTime","Flags-flagPercentages","Flags-flagPercentageBar","Sidebar-hideCommunityBulletin","Sidebar-hideJustHotMetaPosts","Sidebar-hideHireMe","Sidebar-hideLoveThisSite","Sidebar-hideHotNetworkQuestions","Sidebar-linkedToFrom","Sidebar-hotNetworkQuestionsFiltering","Chat-chatEasyAccess","Chat-replyToOwnChatMessages","Chat-renameChat","Extras-alwaysShowImageUploadLinkBox","Extras-linkedPostsInline","Extras-parseCrossSiteLinks","Extras-quickAuthorInfo","Extras-shareLinksMarkdown","Extras-sortByBountyAmount","Extras-copyCode","Extras-dailyReviewBar","Extras-openLinksInNewTab","Extras-showQuestionStateInSuggestedEditReviewQueue","Extras-addOnTopicLinkToSiteSwitcher","Extras-customMagicLinks"]
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  downvotedPostsEditAlert
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {"112350":{"questionId":"112350","addedDate":1531107411248,"sitename":"academia","lastCheckedTime":1532447213778}}
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  editReasons
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  [{"formatting":"Improved Formatting"},{"spelling":"Corrected Spelling"},{"grammar":"Fixed grammar"},{"greetings":"Removed thanks/greetings"},{"tag":"Added tags"}]
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  metaNewQuestionAlert-lastQuestions
userscript.html?id=e74b9685-9ae2-4ffd-b3e4-65434d01587e:68 SOX:  {}
```

**Comment by shu8** [comment]
Are you logged in when this happens? I can't ever see the 'love this site' module even when I disable SOX!

If you're not logged in, are you logged in on another SE site? (just trying to figure out who that box appears to!)

**Comment by ooker777** [comment]
Yes, I login in all site with my account "Ooker". Maybe for longtime users the module doesn't show up?

**Comment by shu8** [comment]
@ooker777 thanks, you were right, I managed to reproduce on an account that was new to a site. Sorry this took so long, but should be fixed in [dev 2.4.4](https://github.com/soscripted/sox/raw/dev/sox.user.js)! :)
